### ISSUE TITLE
Language problem

### ISSUE THREAD

**Issue by littlecxm** [kwsch/NHSE#140]
**Is your feature request related to a problem? Please describe.**
- NHSE.WinForms/Util/DevUtil.cs
    1. Crash on generate lang list before `main.data` load
- NHSE.WinForms/Util/MessageStrings.cs
    1. Not work on other language except integrate english

**Describe the solution you'd like**
- DevUtil.cs
    - Move debug press key on `main.data` loaded
- etc.
    - Add other sting to lang list like `PropertyGrid` & `B_Open`(Main.cs) content

**Comment by kwsch** [comment]
DevUtil is mostly for me to just run periodically to backfill the program translation resources.

Here's the process I use to get the resources updated:

1. Open the program
2. Open a save file
3. Ensure the current language is English. If the language is not english, close the program entirely and start from 1.
4. Close the save file Editor window
5. Press the hotkey from the main window
6. Assets get updated; program closes when finished.
7. Push commit with updated strings

**Comment by kwsch** [comment]
Apologies if it's a little hard to follow; I had originally developed the translation strategy for PKHeX, and copied it over to NHSE. The logic is pretty rigid, since it never needed to be flexible.

**Comment by littlecxm** [comment]
got it👍

**Comment by littlecxm** [comment]
sorry for bother again, but `MessageStrings` still not work? i cant see it initialization any place like `WinFormsTranslator`
